portal-frontend/src/app/features/notice-of-intents/notice-of-intent-details/parcel/parcel.component.html

```diff
@@ -5,7 +5,7 @@ <h3 class="flex-item">1. Identify Parcel(s) Under Application</h3>
       <h4>Parcel #{{ parcelInd + 1 }}</h4>
       <div>
         <button
-          *ngIf="showEdit"
+          *ngIf="showEdit && parcels.length > 1"
```

Comment:
`showEdit` is just set `true` and never changed so doesn't really need to exist. I think either just removing it all together or moving `parcels.length > 1` into .ts file and setting `shoeEdit` based on it.